spec/command/plugins/create_spec.rb

```diff
@@ -7,8 +7,21 @@ module CLAide
   describe Command::Plugins::Create do
     extend SpecHelper::PluginsCreateCommand
 
+    after do
+      config =  CLAide::Plugins::Configuration.new('CLAideTesting',
```

Comment:
The CLAideTesting config could be exposed as a reference/method from the plugins_helper so this does not have to be copied to here